Agregar paginación a Miembros

MacaHannaArenas opened the pull request:
![imagen](https://user-images.githubusercontent.com/92378693/175794017-c5e174d4-a3f4-4247-9e5a-cb302be2f33d.png)

MacaHannaArenas committed:
Agregar paginación a Miembros

MacaHannaArenas committed:
Agregar paginación a Miembros

turcoamado commented on src/main/java/com/alkemy/ong/domain/service/impl/MemberServiceImpl.java:
```diff
@@ -59,4 +69,26 @@ public List<MemberDTO> getAll() {
+    public PageDTO<MemberDTO> getAllMembersPageable(Integer page) {
+        PageDTO<MemberDTO> memberDTOPageDTO = new PageDTO<>();
+        Page<Member> member = this.memberRepository.findAll(PageRequest.of(page - 1, Url.MAX_PAGE, Sort.by("name")));
+        if (member.isEmpty()) {
```
Maca estos if creo que estaban en un Util. Fijate la clase Url.java que esta ahi dentro, podes llamar a eso en lugar de codearlos aca. En la clase _NewsServiceImpl_ hay un ejemplo

MacaHannaArenas replied:
gracias!

MacaHannaArenas replied:
Ahi lo arregle

turcoamado reviewed (changes requested):
Fijate eso que te mencione para reutilizar la clase Url dentro de utils. Lo otro esta bien

MacaHannaArenas committed:
Merge remote-tracking branch 'Git_Alkemy/develop' into feature/OT207-82_PaginaciónMiembros

# Conflicts:
#	src/main/java/com/alkemy/ong/domain/repository/CommentaryRepository.java
#	src/main/java/com/alkemy/ong/domain/service/ICommentaryService.java
#	src/main/java/com/alkemy/ong/domain/util/Url.java

MacaHannaArenas committed:
Utilizacion de url class para hacer comprobaciones

MacaHannaArenas committed:
java: cannot find symbol
  symbol:   variable newsRepository
  location: class com.alkemy.ong.domain.service.impl.CommentaryServiceImpl

MacaHannaArenas committed:
Merge remote-tracking branch 'Git_Alkemy/develop' into feature/OT207-82_PaginaciónMiembros

MacaHannaArenas committed:
newsRepository  en Commentary Service

MacaHannaArenas commented:
![imagen](https://user-images.githubusercontent.com/92378693/175980855-cb635d8f-0c17-4847-9a8e-7626639f0853.png)
![imagen](https://user-images.githubusercontent.com/92378693/175980908-27c4348e-41dc-4af7-8b1c-f1e198cce1b5.png)
